vowpalwabbit/cost_sensitive.cc

```diff
@@ -226,13 +226,13 @@ void print_update(vw& all, bool is_test, example& ec, multi_ex* ec_seq, bool act
         pred_buf << ec.pred.a_s[0].action;
       if (action_scores)
         pred_buf << ".....";
-      all.sd->print_update(all.holdout_set_off, all.current_pass, label_buf, pred_buf.str(), num_current_features,
-          all.progress_add, all.progress_arg);
+      all.sd->print_update(all.ec.holdout_set_off, all.gs.current_pass, label_buf, pred_buf.str(), num_current_features,
```

Comment:
I know you're planning on changing the names but when you see `ec` in VW, by default you usually think examples collection which is misleading here

Reply:
changed to example_config, to avoid confusion while doing pr. We can hammer down on the final names later.